wp-admin/js/widgets/media-image-widget.js

```diff
@@ -69,6 +88,38 @@
 		},
 
 		/**
+		 * Get the instance props from the media selection frame.
+		 *
+		 * @param {object} attachment Attachment object.
+		 * @returns {object} Props
+		 */
+		_getEmbedProps: function _getAttachmentProps( attachment ) {
+			var props = {};
+
+			if ( ! _.isEmpty( attachment ) ) {
+				props = {
+					attachment_id: attachment.id,
+					align: attachment.align,
+					alt: attachment.alt,
+					caption: attachment.caption,
+					image_classes: '',
+					image_title: '',
+					link_classes: '',
+					link_rel: '',
+					link_url: attachment.linkUrl ? attachment.linkUrl : attachment.link,
```

Comment:
here is the passed data when I edit an embedded image:

![](https://d3vv6lp55qjaqc.cloudfront.net/items/1T2v2m0m1G2F0Y0c413Q/Image%202017-04-04%20at%208.20.21%20AM.png?X-CloudApp-Visitor-Id=1371633)